core/src/main/java/de/danoeh/antennapod/core/util/comparator/InReverseChronologicalOrder.java

```diff
@@ -0,0 +1,18 @@
+package de.danoeh.antennapod.core.util.comparator;
+
+import java.util.Comparator;
+
+import de.danoeh.antennapod.core.feed.FeedItem;
+import de.danoeh.antennapod.core.feed.SearchResult;
+
+public class InReverseChronologicalOrder implements Comparator<SearchResult> {
+    /**
+     * Compare items and sort it on chronological order.
+     */
+    @Override
+    public int compare(SearchResult o1, SearchResult o2) {
+        if (o1.getComponent () instanceof FeedItem && o2.getComponent () instanceof FeedItem)
+            return ((FeedItem) o2.getComponent ()).getPubDate ().compareTo (((FeedItem) o1.getComponent ()).getPubDate ());
```

Comment:
Please add brackets to the `if` statement and break the long line. Also, please remove the spaces between method name and brackets. Sorry for confusing you with my previous comment. We are using spaces between `if` and `(` but not everywhere in front of `(`